pkg/fs/fs_windows.go

```diff
@@ -4,10 +4,28 @@ package fs
 
 import (
 	"os"
+	"os/exec"
 
 	"github.com/hectane/go-acl"
 )
 
 func Chmod(path string, mode os.FileMode) error {
 	return acl.Chmod(path, mode)
 }
+
+func ChmodRecursiveAdmin(path string) error {
+	// Step1: Remove inhereted permissions
```

Comment:
Validation of path is required. Can you bring the check from storage container here

Reply:
Thanks, done